### ISSUE TITLE
Vocaliser le nom ou le numéro appelant via la sortie son du Raspberry Pi

### ISSUE THREAD

**Issue by jystervinou** [jystervinou/freebox-caller-id#6]
Idée : lors d'un appel, il pourrait être intéressant que soit prononcé automatiquement le nom ou le numéro de l'appelant via une enceinte connectée à un raspberry pi sur lequel tourne le script.

Voir aussi la possibilité d'utiliser une [enceinte bluetooth](https://www.raspberrypi.org/magpi/bluetooth-audio-raspberry-pi-3/).

Voir aussi la possibilité d'utiliser le [haut parleur du Freebox Server](http://blog.magiczapper.com/post/92963310876/freebox-56k) ... 👍

**Comment by jystervinou** [comment]
J'ai un prototype qui marche de la dernière version ci-dessus (parce que comme ça, ben y'a pas besoin d'enceinte) :

Le téléphone sonne... le Freebox Server énonce (vocalement donc) le nom de l'appelant (s'il est dans ton carnet d'adresse Freebox) via l'enceinte intégrée.

... donc bientôt dispo.

**Comment by jystervinou** [comment]
"voir possibilité d'utiliser le haut parleur du Freebox Server"

Implémenté dans la version 0.7.2 :-)

**Comment by mikebzh44** [comment]
+1 pour l'enceinte BT. Mon Serveur FB est dans le bureau à l'étage donc pas forcément le plus pratique ;)

Super script en tout cas !!!

**Comment by jystervinou** [comment]
OK reçu pour le +1

Aussi à Nantes ? :)

**Comment by mikebzh44** [comment]
Yes, mais que pour le boulot, sinon, j'habite dans le grand Sud, à la limite du 85 :p

Stervinou, avec un nom comme ça, t'es breton toi aussi :D

**Comment by jystervinou** [comment]
> j'habite dans le grand Sud, à la limite du 85

wow, c'est _wild_ ;-)

> Stervinou, avec un nom comme ça, t'es breton toi aussi 

[oui](https://fr.wiktionary.org/wiki/stêr) et [oui](https://fr.wiktionary.org/wiki/ivinoù)

**Comment by mikebzh44** [comment]
Salut, Rivière d'ongles :p 

En parlant d'enceinte BT, tu penses que ça serait faisable d'utiliser les enceintes connectés style Alexa ou Google Home. Qui à faire une pierre, 2 coups, autant s'équiper d'une Google Home pour piloter la domotique et avoir les notifs des appels ;)

**Comment by jystervinou** [comment]
Roh, plutôt les ifs ;-) quelque-part au bord du canal de Nantes à Brest..

Aucune idée, on peut y diffuser via Bluetooth ou autre protocole ?? Si tu peux regarder ça, ça me défricherait déjà le travail :)

Merci

**Comment by mikebzh44** [comment]
Aucune idée, j'ai pas encore craqué pour ce genre d'enceinte.

Mais on dirait que c'est possible : 

http://www.cnetfrance.fr/news/google-home-le-bluetooth-enfin-disponible-39856252.htm

**Comment by mikebzh44** [comment]
J'ai l'enceinte Google Home Mini maintenant et je suis en train d'installer un autre pluging pour piloter la Freebox et on dirait qu'il permet de faire un retour audio sur le Google Home :

https://aymkdn.github.io/assistant-plugins/?plugin=notifier
